tools/scripts/firehose.py

```diff
@@ -58,40 +59,53 @@
 
 STATUS_OPTIONS = ('successful', 'failed', 'error', 'canceled')
 
+EVENT_OPTIONS = ('runner_on_ok', 'runner_on_failed', 'runner_on_changed', 'runner_on_skipped', 'runner_on_unreachable')
+
+MODULE_OPTIONS = ('yup', 'stonchronize', 'templotz', 'deboog')
 
 class YieldedRows(StringIO):
 
     def __init__(self, job_id, rows, created_stamp, modified_stamp, *args, **kwargs):
         self.rows = rows
-        self.row = "\t".join([
-            f"{created_stamp}",
-            f"{modified_stamp}",
-            "playbook_on_start",
-            "{}",
-            'false',
-            'false',
-            "localhost",
-            "Example Play",
-            "Hello World",
-            "",
-            "0",
-            "1",
-            job_id,
-            u,
-            "",
-            "1",
-            "hello_world.yml",
-            "0",
-            "X",
-            "1",
-        ]) + '\n'
+        self.rowlist = []
+        for i in range(len(EVENT_OPTIONS)):
```

Comment:
You can do this a lot more simply without having to write a nested loop:

```python
import itertools
EVENT_OPTIONS = itertools.cycle([...])
MODULE_OPTIONS = itertools.cycle([...])

i = next(EVENT_OPTIONS)
j = next(MODULE_OPTIONS)
```

Reply:
TIL, thx.